content/300-guides/050-database/890-Neon

```diff
@@ -0,0 +1,131 @@
+---
+title: 'Using Prisma with Neon'
+metaTitle: 'Using Prisma with Neon'
+metaDescription: 'Guide to using Prisma with Neon'
+tocDepth: 2
+toc: true 
+---
+
+<TopBlock>
+
+This guide explains how to connect Prisma to Neon, establish connections when using Prisma Client in serverless functions, and resolve [connection timeout](#connection-timeouts) issues.
+
+To configure Prisma Migrate with Neon, see [Use Prisma Migrate with Neon](../guides/prisma-migrate).
+
+</TopBlock>
+
+## What is Neon?
+
+[Neon](https://neon.tech/) Neon is a fully managed serverless PostgreSQL with a generous free tier. Neon separates storage and compute and offers modern developer features such as serverless, branching, bottomless storage, and more. Neon is open source and written in Rust.
+
+Learn more about Neon [here](https://neon.tech/docs).
+
+## Prerequisites
+
+- A Neon project. See [Create a project](../manage/projects#create-a-project).
+- A Prisma project. See [Set up Prisma](https://www.prisma.io/docs/getting-started/setup-prisma), in the _Prisma documentation_.
+
+## Connect to Neon from Prisma
+
+To establish a basic connection from Prisma to Neon, perform the following steps:
+
+1. Retrieve your Neon connection string. In the **Connection Details** widget on the Neon **Dashboard**, select a branch, a user, and the database you want to connect to. A connection string is constructed for you.
+  ![Connection details widget](/docs/connect/connection_details.png)
+  The connection string includes the user name, password, hostname, and database name.
+
+2. Add the following lines to your `prisma/schema.prisma` file to identify the data source and database URL:
+
+   ```typescript
+   datasource db {
+     provider = "postgresql"
+     url   = env("DATABASE_URL")
+   }
+   ```
+
+3. Add a `DATABASE_URL` variable to your `.env` file and set it to the Neon connection string that you copied in the previous step. Your setting will appear similar to the following:
+
+   <CodeBlock shouldWrap>
+
+   ```text
+   DATABASE_URL="postgres://daniel:<password>@ep-mute-rain-952417.us-east-2.aws.neon.tech/neondb"
+   ```
+
+   </CodeBlock>
+
+<Admonition type="important">
+If you are using Prisma Client from a serverless function, see [Connect from serverless functions](#connect-from-serverless-functions). To adjust your connection string to avoid connection timeouts issues, see [Connection timeouts](#connection-timeouts).
+</Admonition>
+
+## Connect from serverless functions
```

Comment:
In this section, we could also borrow from the structure used in the Specific considerations section in the "Using Prisma with Supabase" guide. 

We would still distinguish between the two connection strings (pooled and non-pooled) but recommend using the `directUrl` property in the `datasource` block.

This is similar to how it's represented in the Neon docs here: https://neon.tech/docs/guides/prisma-migrate#prisma-migrate-with-pgbouncer